From af14249f241737e02f202a0a697bf8b0cbca2431 Mon Sep 17 00:00:00 2001
From: Fancy Fang <B47543@freescale.com>
Date: Fri, 20 Dec 2013 10:14:06 +0800
Subject: [PATCH 247/249] ENGR00292900 PXP: change pxp library to match kernel
 modification

After the linux kernel commit "ENGR00292562 PXP: move the
definitions used only by PXP device to a new header file",
pxp library should include the new header file pxp_device.h
instead of pxp_dma.h any more.

Signed-off-by: Fancy Fang <B47543@freescale.com>
---
 pxp/pxp_lib.h | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/pxp/pxp_lib.h b/pxp/pxp_lib.h
index ae4aed5..1c023af 100644
--- a/pxp/pxp_lib.h
+++ b/pxp/pxp_lib.h
@@ -1,7 +1,7 @@
 /*
  * pxp_lib - a user space library for PxP
  *
- * Copyright (C) 2010 Freescale Semiconductor, Inc.
+ * Copyright (C) 2010-2013 Freescale Semiconductor, Inc.
  */
 
 /*
@@ -16,7 +16,7 @@
 #ifndef	PXP_LIB_H
 #define	PXP_LIB_H
 
-#include <linux/pxp_dma.h>
+#include <linux/pxp_device.h>
 
 #ifndef true
 #define true    1
@@ -25,10 +25,7 @@
 #define false   0
 #endif
 
-typedef struct pxp_chan_handle {
-	int chan_id;
-	int hist_status;
-} pxp_chan_handle_t;
+typedef struct pxp_chan_handle pxp_chan_handle_t;
 
 int pxp_init();
 void pxp_uninit();
-- 
1.8.0

